ccmlib/cluster.py

```diff
@@ -421,7 +424,7 @@ def remove_dir_with_retry(self, path):
                 try:
                     common.rmdirs(path)
                     removed = True
-                except:
+                except Exception:
```

Comment:
Linter should be angry about this, isn't it ? too broad exception ?

I think in this case we can set a specific set of possible exceptions